Dockerfile

```diff
@@ -39,6 +40,13 @@ RUN apk add --no-cache git \
 WORKDIR /app
 RUN CGO_ENABLED=0 GOOS=linux go build
 
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+# Install https://docs.asciidoctor.org/diagram-extension/latest/diagram_types/a2s/
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+FROM golang:${A2S_GOLANG_VERSION}-alpine as a2s-builder
+ARG A2S_VERSION=ca82a5c
```

Comment:
asciitosvg repo isn't tagged. I chose to base the Dockerfile on main's current SHA so that any updates are made deliberately  and container builds have repeatable content.

Reply:
```suggestion
# Expects a git reference as there are no tags in the A2S repository
ARG A2S_VERSION=ca82a5c
```

Thanks for the explanation, it make sense. WDYt about a comment here until we can track the reference with `updatecli` ?